src/number_theory/bernoulli_polynomials.lean

```diff
@@ -43,11 +42,13 @@ open_locale nat
 
 open nat finset
 
+namespace polynomial
+
 /-- The Bernoulli polynomials are defined in terms of the negative Bernoulli numbers. -/
 def bernoulli_poly (n : ℕ) : polynomial ℚ :=
```

Comment:
As we are going to be inside the `polynomial` namespace here i think this can just be `bernoulli` instead of `bernoulli_poly` however this will result in collisions with the `bernoulli` in the `number_theory.bernoully` file.

Reply:
Not really, being in the `polynomial` namespace means this will be called `polynomial.bernoulli`, so no collisions.

Reply:
If we do this change there are going to be errors with lines like:

https://github.com/leanprover-community/mathlib/blob/d421e0e535f066795c1bb167daa4c5a8d16ca1f4/src/number_theory/bernoulli_polynomials.lean#L52

This is because there is already a `beurnolli` in the namespace so we will have to do something different in the import maybe ?

Reply:
It was solved by the use of `_root_` in https://github.com/leanprover-community/mathlib/pull/11805/commits/3c7f01d6dac90a3ae13d570a851614709806668c